src/javascript/Edit/Edit.actions.js

```diff
@@ -9,12 +9,22 @@ import OpenWorkInProgressModalAction from '~/EditPanel/WorkInProgress/OpenWorkIn
 import {CopyLanguageActionComponent} from '~/Edit/copyLanguage/copyLanguage.action';
 
 export const registerEditActions = actionsRegistry => {
-    // Edit action button in JContent
+    // Edit action button in JContent; need separate actions for content and pages
     actionsRegistry.add('action', 'edit', editContentAction, {
         buttonIcon: <Edit/>,
         buttonLabel: 'content-editor:label.contentEditor.edit.contentEdit',
         targets: ['contentActions:2'],
-        hideOnNodeTypes: ['jnt:virtualsite'],
+        hideOnNodeTypes: ['jnt:virtualsite', 'jnt:page'], // For edit content
```

Comment:
Just curious why is this added

Reply:
jnt:page is already included in editPage action. need to hide here so we don't get duplicates.